packages/components/src/radio-control/index.tsx

```diff
@@ -16,6 +16,16 @@ import BaseControl from '../base-control';
 import type { WordPressComponentProps } from '../context';
 import type { RadioControlProps } from './types';
 import { VStack } from '../v-stack';
+import { StyledHelp } from '../base-control/styles/base-control-styles';
+
+// This is the id that BaseControl assigns to the help text element.
+function generateHelpTextId( id: string ) {
+	return `${ id }__help`;
+}
```

Comment:
This is hacky, but it's already present on `trunk`. Ideally `BaseControl` should be more flexible.

Reply:
Good point! I used it in https://github.com/WordPress/gutenberg/pull/63751/commits/4a6087ada724ebc5da01334a9f4b714bb5a234f3 and it seems to work well